### ISSUE TITLE
Update rad2deg

### ISSUE THREAD

**Issue by kgryte** [compute-io/todo#56]
[rad2deg](https://github.com/compute-io/rad2deg)

**Comment by Planeshifter** [comment]
Shall we handle the mutation differently, with `copy` option defaulting to `true`?

**Comment by kgryte** [comment]
Yes, this is my current opinion. Functions which has previously mutated by default are now `copy=true` by default; e.g., `erf`, etc.
